### ISSUE TITLE
Site description being cut-off in some cases.

### ISSUE THREAD

**Issue by iamtakashi** [WordPress/twentysixteen#454]
Originally reported by @emiluzelac with #452. The bottom of site description is cut-off: http://envato.d.pr/171h0/5HHngCXz

This is a side effect of #441, specifically `overflow: hidden`. Good thing is that since then max size of custom logo has been changed, so #440 could be not an issue anymore. We still need to test but we possibly can revert #441 to fix the issue.

**Comment by karmatosed** [comment]
@emiluzelac are you looking at fix to close this with a PR?

**Comment by emiluzelac** [comment]
@karmatosed Yeah, the https://github.com/WordPress/twentysixteen/pull/452/files is the fix, should I proceed with that?
